notebooks/Tracking_drift_check.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from scipy.optimize import curve_fit
</pre><p>I believe it would be better to move all the imports to the beginning of the notebook. Could you do that? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>